src/core.ts

```diff
@@ -135,7 +135,7 @@ function envelope(): IEnvelope {
   return {
     clarityId: cid,
     impressionId,
-    url: top.location.href,
+    url: location.href,
```

Comment:
When the page is loaded in the iframe, 'top.location' will try to access parent page's url, which is forbidden by browser security policies (unless both pages are from the same origin), so it causes a javascript error.